docs/UserGuide.md

```diff
@@ -124,7 +125,7 @@ Examples:
 
 ### Listing all persons : `list`
```

Comment:
Can't seem to edit the "Examples" section for the List Command.

But I was thinking perhaps it may be better for Line 142
``* `list` Lists all persons without sorting them.``
To be
``* `list` Lists all persons in the order they were inserted.``